examples/gocean/eg7/Makefile

```diff
@@ -44,10 +44,6 @@
 
 include ../../common.mk
 
-# Reset the default goal to be 'compile', i.e. to include compilation
-.DEFAULT_GOAL:=
-default: compile
```

Comment:
Is this deliberate - i.e. do we now want the default target to be `transform`?

Reply:
Unless I am confused (that PR was a while ago) - didn't we agree that default should be transform everywhere? And 'compile' needs to be explicitly specified? I did notice that ``tutorial/practicals`` still had 'compile' as default, and I've fixed this as well, If we want to have ``compile`` being the default, let me know.

Reply:
I can't remember :-) Having the default as transform seems sensible.